buildspecs/core.feature

```diff
@@ -44,6 +44,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<project id="runtime"/>
 	</source>
 	<flags>
+		<flag id="build_jitserver" value="false"/>
```

Comment:
Please remove this line.  The flag defaults to false so this is redundant.

Reply:
This is required. openjdk-openj9 builds rely on turning `value="false"` to `value="true"` when `--enable-jitserver` is specified. As an example, see the change in jdk11 extensions repo here: https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/239/files.
Without this flag openjdk-openj9 builds won't be able to turn on jitserver support in OpenJ9 VM.

Reply:
This is an unfortunate design pattern but given there's precedent for it, I remove my objection